includes/path.h

```diff
@@ -0,0 +1,10 @@
+#ifndef PATH_H
+# define PATH_H
+
+#include "libft.h"
+
+t_bool	is_in_path(const char *execname);
+
+char	*abspath(const char *execname);
```

Comment:
better for understudning to have :  abs_path or absolute _path

Reply:
abspath is a standard name in many languages like Python and PHP to get  absolute path, stick to conventions?